packages/webapp/src/components/Task/PureIrrigationTask/index.tsx

```diff
@@ -0,0 +1,275 @@
+import React, { FC, useState } from 'react';
+import { Label, Main, Underlined } from '../../Typography';
+import { useTranslation } from 'react-i18next';
+import Button from '../../Form/Button';
+import Form from '../../Form';
+import { Controller, useForm } from 'react-hook-form';
+import MultiStepPageTitle from '../../PageTitle/MultiStepPageTitle';
+import ReactSelect from '../../Form/ReactSelect';
+import Checkbox from '../../Form/Checkbox';
+import RadioGroup from '../../Form/RadioGroup';
+import styles from '../../Typography/typography.module.scss';
+import Input from '../../Form/Input';
+import InputAutoSize from '../../Form/InputAutoSize';
+import WaterUseCalculatorModal from '../../Modals/WaterUseCalculatorModal';
+import Unit from '../../Form/Unit';
+import { waterUsage } from '../../../util/convert-units/unit';
+import CancelFlowModal from '../../Modals/CancelFlowModal';
+
+export interface ISystem {
+  metric: {
+    units: string[];
+    defaultUnit: string;
+    breakpoints: number[];
+  };
+  imperial: {
+    units: string[];
+    defaultUnit: string;
+    breakpoints: number[];
+  };
+}
+export interface IPureIrrigationTask {
+  handleGoBack: () => void;
+  handleContinue: () => void;
+  system: ISystem;
+}
+
+const PureIrrigationTask: FC<IPureIrrigationTask> = ({
+  handleGoBack,
```

Comment:
Please add propTypes for validation

Reply:
@anant00729 with typescript, propTypes aren't required. Both typescript types/interfaces work the same as propTypes.  Typescript does validation for type checking.

Reply:
got it